autode/wrappers/keywords.py

```diff
@@ -128,14 +202,14 @@
     def _string(self, prefix):
         """Return a string defining the keywords, with or without a prefix"""
```

Comment:
this local import doesn't seem to get used?

Reply:
excellent point.. no idea why that is there 🙃